port_set.go

```diff
@@ -51,7 +51,13 @@ func (portSet *PortSettingCommand) Run(args *GlobalOptions) error {
 
 		portSetting := settings[switchPort-1]
 
-		name, err := comparePortSettings(Name, portSetting.Name, portSet.Name)
+		// If the port name was not set by the user, set it to the existing name (otherwise an empty port name is always considered to be the
+		// "new" value which blanks the port name on the setting next update)
+		if portSet.Name == nil {
```

Comment:
Got your point.

Just reading about the changes in this PR... I do understand, the classical problem of "" (blank string) vs. 'nil' (not provided).

So you made the name a pointer variable, which gives us the possibility to use/interpret 'nil' as the intent to not alter the name on setting the next update.

Since this is a "classic" requirement, the only other option is, to create a new internal bool variable "name_provided_by_user", which is set to True, only when the CLI prameter '-n' is given.

I'm fine with the pointer approach.

Reply:
Thanks Martin. I came to the same conclusions. The pointer method seems to work well enough with the limited information passed into `port_set`, but will I look forward to seeing it be refactored at some point. Appreciate the extra set of eyes and review. :+1: